deps/wazuh_testing/wazuh_testing/tools/migration_tool/utils.py

```diff
@@ -9,15 +9,17 @@
 import subprocess as sbp
 
 import mysql.connector
+import wazuh_testing.tools.migration_tool as migration_tool
 from jsonschema import validate
 from jsonschema.exceptions import ValidationError
 from mysql.connector import errorcode
 from wazuh_testing.tools.migration_tool import MIGRATION_TOOL_PATH, CVE5_SCHEMA_PATH, DELTA_SCHEMA_PATH, \
                                                ERROR_MESSAGES, SNAPSHOTS_DIR, DOWNLOADS_DIR, MIGRATION_TOOL_LOG_PATH, \
-                                               MYSQL_CREDENTIALS, UNCOMPRESSED_DIR
```

Comment:
Because in Python, when you import a variable from another module and you change its value in another one, that module that import the variable will have a symbol and not its real actual value (ref: https://stackoverflow.com/a/3536638)